src/lib.rs

```diff
@@ -1684,6 +1684,84 @@ pub mod gl {
                 },
             }
         }
+
+        pub fn gen_samplers(&self, n: GLsizei) -> Vec<GLuint> {
+            let mut result = vec![0 as GLuint; n as usize];
+            match self {
+                Gl::Gl(gl) => unsafe { gl.GenSamplers(n, result.as_mut_ptr()) },
+                Gl::Gles(gles) => unsafe { gles.GenSamplers(n, result.as_mut_ptr()) },
+            };
+            result
+        }
+
+        pub fn delete_samplers(&self, samplers: &[GLuint]) {
+            match self {
+                Gl::Gl(gl) => unsafe { gl.DeleteSamplers(samplers.len() as GLsizei, samplers.as_ptr()) },
+                Gl::Gles(gles) => unsafe { gles.DeleteSamplers(samplers.len() as GLsizei, samplers.as_ptr()) },
+            }
+        }
+
+        pub fn is_sampler(&self, sampler: GLuint) -> GLboolean {
```

Comment:
Let's make this return a book like other is_foo methods.

Reply:
Whoops, thanks!